README.md

```diff
@@ -31,13 +31,17 @@
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 PERSONAL INVENTORY <a name="about-project"></a>
+# 📖 RAILS BLOG APP <a name="about-project"></a>
 
 
 **RAILS BLOG APP** This Blog app represents an engaging blog platform developed with Ruby on Rails. This platform empowers users to generate, peruse, engage with comments, and show appreciation through likes on posts. It stands as a straightforward yet impactful illustration of crafting web applications.
 
-The database schema for this was generated and guided using the UML Diagram below
-![Alt text](image.png)
+WIREFRAME
+![image](https://github.com/geekelo/rails-blog_app/assets/102464578/d223e0ec-551d-4389-a45a-b9e0b62e770c)
+![image](https://github.com/geekelo/rails-blog_app/assets/102464578/b6d75d35-5613-4fec-a23f-533ac2c40af6)
+![image](https://github.com/geekelo/rails-blog_app/assets/102464578/072ef998-2566-4116-a21c-edd58af433da)
+![image](https://github.com/geekelo/rails-blog_app/assets/102464578/8e31087f-f384-46f0-9f9f-befc87ebd8b1)
+
 
```

Comment:
---

Hi @geekelo,

First off, great job on your project so far! The effort and dedication you've put into it is evident. 👏

However, while reviewing, I noticed that a change requested by the previous reviewer concerning the MIT license in your README hasn't been addressed.

**Change Requested:**
The MIT license in your README appears to still be in its template form. It's essential for the license to be customized and tailored to your project. This not only adds a professional touch but also grants users the proper legal authority to use your code.

Please take a moment to update this section to reflect the appropriate details for your project.

Keep pushing forward, and let's make this adjustment to proceed!

---

Happy Coding! 🚀👨‍💻👩‍💻